src/symmetricDifference.js

```diff
@@ -0,0 +1,25 @@
+var _curry2 = require('./internal/_curry2');
+var difference = require('./difference');
+var union = require('./union');
+
+
+/**
+ * Finds the set (i.e. no duplicates) of all elements contained in the first or
+ * second list, but not both.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig [a] -> [a] -> [a]
```

Comment:
Perhaps this should be `[*] -> [*] -> [*]`, since there's no requirement that the lists' values be of a single type.